frontend/src/Commons/Forms/InputModules/FieldsAndSoil/FieldsAndSoil.tsx

```diff
@@ -93,16 +98,16 @@ const FieldsAndSoilComponent: FC<InputModuleProps> = ({
         FieldName: values.FieldName,
         Area: values.Area,
         Comment: values.Comment,
-        hasSoilTest: values.hasSoilTest,
+        HasSoilTest: values.HasSoilTest,
         SoilTest: {
           TestingMethod: values.SoilTest.TestingMethod,
           sampleDate: values.SoilTest.sampleDate,
           valNO3H: values.SoilTest.valNO3H,
-          valP: values.SoilTest.valP,
+          ValP: values.SoilTest.ValP,
```

Comment:
I believe this might be a typo from .nmp files, where they have an uppercase for this var but not the rest. We can find a better solution in the future, for now this is the quick and dirty way to have everything load.